test/common/formatter/substitution_formatter_test.cc

```diff
@@ -841,6 +841,18 @@ TEST(SubstitutionFormatterTest, streamInfoFormatter) {
                 ProtoEq(ValueUtil::stringValue("127.0.0.2:0")));
   }
 
+  {
+    StreamInfoFormatter upstream_format("DOWNSTREAM_DIRECT_LOCAL_ADDRESS");
```

Comment:
Nit: Copy-paste? Seems like this doesn't have to do with the upstream. Could it and the other lines below be named accordingly?

Reply:
I renamed it for format